modules/ROOT/pages/index.adoc

```diff
@@ -8,26 +15,129 @@
 
 :console: 0
 
-This is the Neo4j Operations Manual, which includes all the operational details and instructions for installing and deploying Neo4j on-premise and in the cloud.
+[.display.cards]
 
-For all information on *upgrading and migrating Neo4j*, see the link:{neo4j-docs-base-uri}/upgrade-migration-guide/current/[Neo4j Upgrade and Migration Guide].
+== Install, deploy and operate Neo4j in a self-managed environment
```

Comment:
Looks much better now.
However, I still think that having an intro sentence looks better than starting with "For more information...". If we have to remove something, then let's remove these two sentences:
"For information on upgrading and migrating Neo4j, see the link:https://neo4j.com/docs/upgrade-migration-guide/[Neo4j Upgrade and Migration] documentation.
For link:https://neo4j.com/aura/[Aura], the Neo4j fully managed cloud service, see the link:https://neo4j.com/docs/aura[Neo4j Aura] documentation."

Reply:
You mean keeping only the info about the current Neo4j version then?

Reply:
No, I mean removing this heading: Install, deploy and operate Neo4j in a self-managed environment and having a sentence instead like I already suggested:


```suggestion
Neo4j Operations includes all the operational details and instructions for installing, deploying, and operating Neo4j in a self-managed environment. 

The latest version of Neo4j is *{neo4j-version-exact}*.
```

Reply:
But who created this layout? Can't we amend it to suit our needs? Or we can have slightly different layouts for different needs/manuals?

Reply:
Sebastian from the design team created this, DocOps executes it. We can make amends when necessary for a functional reason, which I believe it's not the case here. We need to make sure we follow a standard and create consistency across all docsets.